src/serverenvironment.cpp

```diff
@@ -1926,9 +1921,16 @@ u16 ServerEnvironment::addActiveObjectRaw(std::unique_ptr<ServerActiveObject> ob
 			errorstream<<"ServerEnvironment::addActiveObjectRaw(): "
```

Comment:
So we never actually hit this branch before? (Because ServerMap::emergeBlock by default will create a blank block.)

Reply:
Resolved based on sfan5 comment below.